File: Java/CarroTipoC.java
import java.util.ArrayList;
import java.util.Map;

public class CarroTipoC extends Carro{

    Map<String, Map<String, Integer>> tipoCarroAceptado;
    ArrayList<String> materialAsientos;

    public CarroTipoC(Cuenta driver, String license, 
            Map<String, Map<String, Integer>> tipoCarroAceptado, ArrayList<String> materialAsientos) {
        super(driver, license);
        this.tipoCarroAceptado = tipoCarroAceptado;
        this.materialAsientos = materialAsientos;
    }
    
}

File: Python/carroTipoC.py
from carro import Carro
class carroTipoA(Carro):
    tipoCarroAceptado = []
    materialAsientos = []

    def __init__(self, license, driver, tipoCarroAceptado,  materialAsientos):
        super.__init__(license, driver)
        self.tipoCarroAceptado = tipoCarroAceptado
        self.materialAsientos = materialAsientos